package.json

```diff
@@ -26,6 +26,10 @@
     "publish": "ts-node --project ./scripts/tsconfig.json ./scripts/release/Release.ts",
     "release": "lerna run release --ignore @mendix/custom-widgets-utils-internal",
     "version": "ts-node --project ./scripts/tsconfig.json ./scripts/release/BumpVersion.ts",
+    "release:native": "npm run release -- --scope '*-native'",
+    "release:web": "npm run release -- --scope '*-web'",
+    "version": "ts-node --project ./scripts/tsconfig.json ./scripts/release/BumpVersion.ts",
+    "validate-staged-widget-versions": "node scripts/validation/validate-versions-staged-files.js"
```

Comment:
Missing end-of-line comma

Reply:
Yes, it was lost during resolving conflict directly on a web browser. Fixed it now ✔️